src/style.css

```diff
@@ -105,9 +105,11 @@ a {
 
 /* Styles for content rendered by marked (also used by react-simplemde-editor) */
 .markdown-body a, .simplemde-container .editor-preview a {
-  color: #4d72aa;
+  color: #4D72AA;
```

Comment:
it is interesting that we do all caps sometimes not otherwise, but they are the same shade.
I'm looking around at different use-cases in the code.

I feel like we should just stick to one but I'm not sure which one

Reply:
Looking for any thoughts on this!